Project: Milestone1

taiwocoker opened the pull request:
## In this milestone, we added:
-lib folder
-bin folder which has the main file
-linter files

imhilla committed:
add initial files for tictactoe

taiwocoker committed:
Update main.rb

taiwocoker committed:
Update main.rb

imhilla committed:
add content for player 2

taiwocoker committed:
fixed linter error

imhilla committed:
Update main.rb

imhilla committed:
Merge branch 'user_interface' of https://github.com/taiwocoker/Tic-Tac-Toe into user_interface

imhilla committed:
Update main.rb

taiwocoker committed:
Update main.rb

imhilla committed:
Update main.rb

imhilla committed:
Merge branch 'user_interface' of https://github.com/taiwocoker/Tic-Tac-Toe into user_interface

uryelah commented:
Hello **Taiwo Coker & Hillary Odhiambo**, I'm Sarah, your reviewer for this PR 👩‍🚀  

Good job setting up your project!

There's still just a requirement missing though, check the comments below 🦾 

--------

## Changes Required 🏗️ 🐛 💡

- [ ] Remember to add the magic line at the top of the bin/main.rb file: `#!/usr/bin/env ruby` to make it an executable file.

- [ ] As requested by pathwright, print "Hello world" under the `#!/usr/bin/env ruby` and test if the executable file works (bin/main.rb).
![execute_bin](https://user-images.githubusercontent.com/16193212/82448085-a354f700-9a7f-11ea-93a1-143264f6ec24.png)


-----

 **Kindly fix the pointed out issues before requesting the next code review**

------------

Cheers and Happy coding!👏👏👏

Code review by **Sarah Uryelah Silva Chamorro**

(Feel free to contact me on slack by the handle of  **sarah** if you have any additional question)

taiwocoker committed:
Update main.rb

taiwocoker committed:
Update main.rb

taiwocoker committed:
Update main.rb

taiwocoker committed:
Update main.rb

taiwocoker committed:
Update main.rb

pelzolga123 reviewed (approved):
Approved.

pelzolga123 commented:
## Great work, Congratulation :+1: Milestone Approved.
You did all changes required previously. Go ahead and merge to develop.
_Goodluck in your next milestone_ *Happy coding*

imhilla committed:
Update main.rb

taiwocoker committed:
Update main.rb

imhilla committed:
Update main.rb

taiwocoker committed:
Update main.rb

taiwocoker committed:
fixed the check for game won and linters

imhilla committed:
Merge branch 'user_interface' of https://github.com/taiwocoker/Tic-Tac-Toe into user_interface

imhilla committed:
add files and initialize a board

imhilla committed:
add win criteria on the board

imhilla committed:
add puts to display board

taiwocoker committed:
add move methods

taiwocoker committed:
add methods for player won and full board

taiwocoker committed:
add method for position to integer

imhilla committed:
initialize validate class

append boad
initialize the validate
add valid options
add selected options
add valid input to check the positions
update the selected positon

imhilla committed:
require board and validate

create instance of board and create new c=validation

imhilla committed:
create method to define player's move

take player name and validate the option picked on the bord. Until its valid option update the board

taiwocoker committed:
add loop to check if the board is full

taiwocoker committed:
Update main.rb

imhilla committed:
welcome players

taiwocoker committed:
fixed linter errors

imhilla committed:
add readme

taiwocoker committed:
Update  README file

imhilla committed:
Update README.md

imhilla committed:
solved linter errors

taiwocoker committed:
Fixed the required changes

imhilla committed:
Update README.md

imhilla committed:
Update README.md

taiwocoker committed:
Fixed the Draw method

imhilla committed:
Update README.md

imhilla committed:
Merge branch 'readme_game_instructions' into game_logic

imhilla committed:
Merge branch 'user_interface' into milestone_1
